src/dndbeyond/content-scripts/character.js

```diff
@@ -586,7 +586,9 @@ function handleSpecialWeaponAttacks(damages=[], damage_types=[], properties, set
     if (character.hasClass("Blood Hunter")) {
         // Bloodhunter: Crimson Rite
         if (character.getSetting("bloodhunter-crimson-rite", false) &&
-            character.hasClassFeature("Crimson Rite")) {
+            character.hasClassFeature("Crimson Rite") &&
+            ((properties["Attack Type"] == "Ranged" || properties["Attack Type"] == "Melee") ||
+            action_name.includes("Predatory Strike"))) {
```

Comment:
Humm... why "Ranged || Melee"... what other type of weapon attacks are there?

Reply:
Not sure this is the best solution honestly, but it was a way to say "If it's Ranged or Melee" (rollItem), or the action_name includes "Predatory Strike" (rollAction) then do the thing. Probably a better way, I just couldn't see it at the time.

Reply:
I believe this break also occurred due to an update to Bloodhunter. I mostly just don't like Bloodhunter from all the weird broken crap it appears to have (in the example character sheet I was given, Unarmed Strike has proficiency, Predatory strikes, the "buffed" unarmed strike has no proficiency. It's just a mess of a class).

Reply:
My confusion, I thought this was in `rollItem`, thanks for the explanation.